scripts/brp-strip-comment-note

```diff
@@ -15,11 +15,11 @@ esac
 
 # Strip .comment and .note sections (the latter only if it is not allocated)
 # for already stripped elf files in the build root
-for f in `find "$RPM_BUILD_ROOT" -type f \( -perm -0100 -or -perm -0010 -or -perm -0001 \) -print0 | xargs -0 -r -P$NCPUS -n32 sh -c "file \"\\$@\" | grep -v \"^${RPM_BUILD_ROOT}/\?usr/lib/debug\" | sed -n -e 's/^\(.*\):[ 	]*ELF.*, stripped.*/\1/p'" ARG0`; do
+find "$RPM_BUILD_ROOT" -type f \( -perm -0100 -or -perm -0010 -or -perm -0001 \) -print0 | xargs -0 -r -P$NCPUS -n32 sh -c 'for f in $(file $@ | grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug" | sed -n -e "s/^\(.*\):[ 	]*ELF.*, stripped.*/\1/p"); do
```

Comment:
I'd also put single quotes around the `${RPM_BUILD_ROOT}` variable in the grep command. It won't change much (other than expanding the variable only once, when calling this script) but will make it more obvious which variables are "static".

Reply:
Hmm, thinking about it again, scratch that. Expanding the variable beforehand (and passing the expanded path to the grep command) could be risky in case the expanded path happens to contain a dollar sign, which `sh -c` would then interpret as variable expansion.